docs/read-folder-tool.md

```diff
@@ -11,3 +11,31 @@ Compose diograph.json from folder contents
 1. Read image EXIF-data
 1. Compose diograph object/json
 1. Save diograph.json file
+
+
+## Open questions
+
+### Diory
+### Id
+- id can be anything
+- uuid
+- content hash -> ipfs -> new versions using ipns
+- created time, great for timeline
```

Comment:
Do you mean created time as an id? I think "createdTime" attribute would be better for created time. And if timestamp is used at id, I'd still suggest having some random characters in the end to ensure the uniqueness.

Reply:
we need an id that somehow links to the actual file because if and when we read the folder again all manual links should remain. This can be done using a content hash. I was thinking that created time would be easier and simpler but probability for same ids is too high.

Reply:
Aah, now I understand, that's true. Content hash sounds good to me. And I think that's the only viable option that I can think of...